packages/caliper-core/test/worker/tx-observers/prometheus-manager-tx-observer.js

```diff
@@ -0,0 +1,214 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const mockery = require('mockery');
+const sinon = require('sinon');
+
+/**
+ * simulate Util
+ */
+class Utils {
+    /**
+     *
+     * @param {*} path path
+     * @return {string} the fake path
+     */
+    static resolvePath(path) {
+        return 'fake/path';
+    }
+
+    /**
+     *
+     * @return {boolean} the fake path
+     */
+    static isForkedProcess() {
+        return false;
+    }
+
+    /**
+     *
+     * @param {*} yaml res
+     * @return {string} the fake yaml
+     */
+    static parseYaml(yaml) {
+        return 'yaml';
+    }
+
+    /**
+     * @returns {*} logger stub
+     */
+    static getLogger() {
+        return {
+            debug: sinon.stub(),
+            error: sinon.stub()
+        };
+    }
+
+    /**
+     * @param {*} url url
+     * @returns {*} url
+     */
+    static augmentUrlWithBasicAuth(url) {
+        return url;
+    }
+}
+
+mockery.enable({
+    warnOnReplace: false,
+    warnOnUnregistered: false
+});
+mockery.registerMock('../../common/utils/caliper-utils', Utils);
```

Comment:
Is this actually needed ? the class under test doesn't seem to use the Utils package

Reply:
This is needed because the base interface (TxObserverInterface) is dependent on the Utils class. Without the mock, this error would be thrown on running the tests:
```  1) When using a PrometheusManagerTxObserver should set managerUuid passed through constructor:
     Error: Util.resolvePath: Parameter is undefined
      at Function.resolvePath (lib/common/utils/caliper-utils.js:247:19)
      at new TxObserverInterface (lib/worker/tx-observers/tx-observer-interface.js:26:61)
      at new PrometheusManagerTxObserver (lib/worker/tx-observers/prometheus-manager-tx-observer.js:21:105)
      at new createTxObserver (lib/worker/tx-observers/prometheus-manager-tx-observer.js:36:123)
      at Context.<anonymous> (test/worker/tx-observers/prometheus-manager-tx-observer.js:90:26)
      at processImmediate (node:internal/timers:466:21)```